src/rules/tsUtils.ts

```diff
@@ -75,6 +89,27 @@ export const isExpectCallWithParent = (
   node.parent.type === AST_NODE_TYPES.MemberExpression &&
   node.parent.property.type === AST_NODE_TYPES.Identifier;
 
+/**
+ * Checks if the given `node` is a {@link JestExpectCallWithNotParent}.
+ *
+ * This is any call to `expect` that is followed by the `not` namespace:
+ *
+ * @example```
+ * expect('a').not.toBe('b');
+ * expect().not
+ * ```
+ *
+ * @param {Node} node
+ *
+ * @return {node is JestExpectCallWithNotParent}
+ */
+export const isExpectCallWithNot = (
```

Comment:
could possibly have a version with this that receives a `JestExpectCallWithParent` which just checks the parent? in this rule, we call `isExpectCallWithParent` twice. Probably not a big issue as the checks are pretty cheap